docs/mvc/tutorials/mvc-with-entity-framework.rst

```diff
@@ -1,8 +1,357 @@
+
+.. note:: This tutorial uses Visual Studio 2015. If you are completely new to ASP.NET MVC or Visual Studio, read :doc:`../getting-started/first-mvc-app` first.
+
+The sample app that you'll build manages a list of authors and books. Here is a screenshot of the app:
+
+.. image:: mvc-with-entity-framework/_static/screenshot1.png
+
+The app uses Razor to generate static HTML. (An alternate approach is to update pages dynamically on the client, with a combination of AJAX calls, JSON data, and client-side JavaScript. This tutorial doesn’t cover that approach.)  
+
+In this article:
+
+    - `Create the project`_
+    - `Add Entity Framework`_
+    - `Create models`_
+    - `Add a DbContext class`_
+    - `Configure Entity Framework`_
+    - `Add an index page`_
+    - `Add a details page`_
+    - `Add a create page`_
+    - `Add an edit page`_
+    - `Add a delete page`_
+    - `Wrapping up`_
+
+Browse the source code for the sample app on `GitHub <https://github.com/aspnet/Docs/tree/master/docs/mvc/tutorials/mvc-with-entity-framework/sample>`_.
+
+
+Create the project
+------------------
+
+Start Visual Studio 2015. From the **File** menu, select **New** > **Project**. 
+
+Select the **ASP.NET Web Application** project template. It appears under **Installed** > **Templates** > **Visual C#** > **Web**. Name the project ``ContosoBooks`` and click **OK**.
+
+.. image:: mvc-with-entity-framework/_static/new-project.png
+
+In the **New Project** dialog, select **Web Site** under **ASP.NET 5 Preview Templates**.
+
+.. image:: mvc-with-entity-framework/_static/web-site-template.png
+
+Click **Change Authentication** and select **No Authentication**. You won’t need authentication for this sample. Click **OK** twice to complete the dialogs and create the project.
+
+.. image:: mvc-with-entity-framework/_static/no-authentication.png
+
+Open the *Views/Shared/_Layout.cshtml* file. Replace the following code:
+
+.. code-block:: aspx-cs
+
+    <li><a asp-controller="Home" asp-action="Index">Home</a></li>
+    <li><a asp-controller="Home" asp-action="About">About</a></li>
+    <li><a asp-controller="Home" asp-action="Contact">Contact</a></li>
+
+with this:
+
+.. literalinclude:: mvc-with-entity-framework/sample/src/ContosoBooks/Views/Shared/_Layout.cshtml
+    :language: aspx-cs
+    :lines: 37-39
+    :dedent: 24
+
+This adds links to the Books and Authors pages (which we haven’t created yet). 
+
+Add Entity Framework
+--------------------
+
+Open the *project.json* file. In the dependencies section, add the following line:
+
+.. code-block:: json
+
+  "dependencies": {
+    ...
+    "EntityFramework.SqlServer": "7.0.0-beta4"
+  },
+
+When you save *project.json*, Visual Studio automatically resolves the new package reference. 
+
+.. image:: mvc-with-entity-framework/_static/package-restore.png
+
+Create models
```

Comment:
"Model" usually refers to the complete model (entity classes + context). I would call this section "Create entity classes" or just "Create entities"... or possibly "Define domain model" (but that is less 101ish)